src/Http.ts

```diff
@@ -85,6 +85,15 @@ export function send<A, Msg>(f: (e: Either<HttpError, A>) => Msg): (req: Request
   return req => xhr(req).pipe(map(result => T.of(O.some(f(result)))))
 }
 
+/**
+ * Executes as `Cmd` the provided call to remote resource, mapping result to a `Msg` containing the full response.
+ * @category utils
+ * @since 0.5.0
+ */
+export function sendWithResp<A, Msg>(f: (e: Either<HttpError, Response<A>>) => Msg): (req: Request<A>) => Cmd<Msg> {
```

Comment:
@bmazzarol this has to be tested (like every other exported/exposed function)

Reply:
No problems will copy over the send tests and give it a spin.